Device/Program.cs

```diff
@@ -14,7 +14,9 @@ public static async Task<int> Main(string[] args)
         {
             var configuration = Configuration.BuildConfiguration();
             var device = new Device(configuration);

```

Comment:
It should have been addressed in earlier PR :) but we should avoid passing configuration directly into the Device class. Device class is only concerned with IoT Hub connection string. Problem with passing entire config is, the Device now has to know about the configuration data structure and it also has access to other configurations even though the Device doesn't necessarily need them.